help/help/Controllers/UsuarioController.cs

```diff
@@ -39,22 +39,39 @@ public ActionResult Cadastrar(UsuarioViewModel userViewModel)
                 Sobrenome = userViewModel.sobreNome,
                 Email = userViewModel.email,
                 Senha = userViewModel.senha,
-                RepeteSenha = userViewModel.repeteSenha
+                Status = false,
+                DataDeCadastro = DateTime.Now,
+                Admin = false
+                
             };
 
-            Usuario userBuscadoNoBanco = _usuarioRepository.BuscarPorEmail(user);
+            Usuario VerificarSeUsuarioExiste = _usuarioRepository.BuscarPorEmail(user);
 
```

Comment:
Não to vendo o código inteiro, creio, porém nessa parte aqui não deveria acontecer ou informar caso o usuário já exista ?

Reply:
Msg de views serão implementadas depois, agora só o cadastramento mesmo !